doc/man-template.roff

```diff
@@ -1,45 +1,148 @@
-[OPTIONS]
-.SS "Positional Arguments"
-.HP
-METHOD
-.IP
-The HTTP method to use. One of \fBget\fR, \fBpost\fR, \fBhead\fR, \fBput\fR, \fBpatch\fR, \fBdelete\fR, or \fBoptions\fR.
-.IP
-Optional. If omitted, a GET or a POST will be done depending on whether the request sends data.
-.HP
-URL
-.IP
-The URL to request. The scheme defaults to 'http://' unless \fB\-\-https\fR is used.
-.IP
-A leading colon works as shorthand for localhost. :8000 is equivalent to localhost:8000, and :/path is equivalent to localhost/path.
-.HP
-REQUEST_ITEM...
-.IP
-Optional key-value pairs to be included in the request.
-.IP
-\fBkey==value\fR to add a parameter to the URL
-.IP
-\fBkey=value\fR to add a JSON field (\fB\-\-json\fR) or form field (\fB\-\-form\fR)
-.IP
-\fBkey=@value\fR to add a JSON field (\fB\-\-json\fR) or form field (\fB\-\-form\fR) from a file.
-.IP
-\fBkey:=value\fR to add a complex JSON value (e.g. `numbers:=[1,2,3]`)
-.IP
-\fBkey:=@value\fR to add a complex JSON value (e.g. `numbers:=@data.json`) from a file.
-.IP
-\fBkey@filename\fR to upload a file from filename (with \fB\-\-form\fR)
-.IP
-\fB@filename\fR to use a file as the request body
-.IP
-\fBheader:value\fR to add a header
-.IP
-\fBheader:\fR to unset a header
-.IP
-\fBheader;\fR to add a header with an empty value
-.IP
-A backslash can be used to escape special characters (e.g. weird\\:key=value).
-.IP
-File uploads can take ';type=' and ';filename=' parameters to set the mimetype and filename, respectively. For example:
-.IP
-  key@filename;type=text/html;filename=document.html
-.SS "Options"
+.TH XH 1 {{date}} {{version}} "User Commands"
+
+.SH NAME
+xh \- Friendly and fast tool for sending HTTP requests
+
+.SH SYNOPSIS
+.B xh
+[\fIOPTIONS\fR]
+[\fIMETHOD\fR]
+\fIURL\fR
+[\-\-\]
+[\fIREQUEST_ITEM\fR ...]
+
+.SH DESCRIPTION
+
+xh is an HTTP client that aims to reimplement as much as possible of
+HTTPie's excellent design, with a focus on improved performance.
```

Comment:
```suggestion
.B xh
is an HTTP client with a friendly command line interface. It strives to
have readable output and easy-to-use options.

xh is mostly compatible with HTTPie: see
.BR http (1).
```
I ultimately couldn't think of a lot to write.

It's common to bold the name at the start of the description, Wikipedia-style, so I added that.

A lot of people won't have the HTTPie man page installed, so the reference could be confusing. But it'll get auto-linkified by man page websites ([e.g.](https://man.archlinux.org/man/xzgrep.1.en)) so on balance I think it's for the best.

Reply:
Thanks, I think this description fits better on a man page.

Would it be helpful to mention the cURL translation feature since that seems to be the feature that users mostly appreciate? or should we avoid pointing to the same feature multiple times on a man page?

Reply:
That might be a good one, since it isn't in HTTPie! I've edited my suggestion.

On a related note, the curl2httpie.online link doesn't work right now, so if it isn't fixed we might have to remove or replace it. I've opened dcb9/curl2httpie#15.

Reply:
>On a related note, the curl2httpie.online link doesn't work right now, so if it isn't fixed we might have to remove or replace it. I've opened https://github.com/dcb9/curl2httpie/issues/15.

I think the certificate for curl2httpie.online has been invalid for quite a while.

If only there were a Rust version that can be compiled into WebAssembly...

Reply:
It's currently a Go version that looks like it's compiled to asm.js, and could be compiled to WebAssembly instead. (It wouldn't be as lean as Rust, but it should be leaner than it is now.)

That makes it possible for us to rehost it, only that seems a little impolite and it might technically be illegal since it doesn't have a license.

Reply:
The website is working again! Maybe we should've opened that issue half a year ago.